classes/publication/PKPPublication.php

```diff
@@ -98,7 +98,7 @@ public function getLocalizedTitle($preferredLocale = null, string $format = 'tex
                 break;
             case 'text':
                 // Title is in HTML, prefix is already in text. Convert title.
-                $title = strip_tags($title);
+                $title = htmlspecialchars_decode(strip_tags($title));
```

Comment:
@bozana I understand the intension and purpose of this change and I think it's ok . However if I am not mistaken and remember correctly, we probably omit the use of `htmlspecialchars_decode` intentionally so that even text version slips into somewhere without the use of `htmlspecialchars` or `strip_unsafe_html`, it will not be abused.  . So developer will be forced to use the `htmlspecialchars_decode` manually as necessary and knowingly . But allowing text version as decoded html special chars, we need to be careful to use `htmlspecialchars` and/or `strip_unsafe_html` if used in template directly . 

But I think it's ok to add this and just need to extra careful of the usage of text version .

Reply:
Hmmm... that sounds reasonable...
@asmecher and @jonasraoni, what do you think?

Reply:
If someone uses a `text` form of the title in the context of an HTML presentation, they'll have to use `escape` (Smarty) or `htmlspecialchars` (PHP)`. But they should use the `html` form instead in that situation. In any case, this is consistent with our plaintext escaping needs/strategy elsewhere in the system, so I'm comfortable with it. :+1: